LdapForNet/LdapEntry.cs

```diff
@@ -218,6 +218,7 @@ public class DirectoryModificationAttribute : DirectoryAttribute
     public class SearchResultAttributeCollection : KeyedCollection<string, DirectoryAttribute>
```

Comment:
Could you please add unit tests for SearchResultAttributeCollection  and ModifyAttributeCollection ?